common/views/components/Modal/Modal.tsx

```diff
@@ -45,9 +45,8 @@ const Overlay = styled.div`
 
 const CloseButton = styled(Space).attrs({
   as: 'button',
-  type: 'button',
-  v: { size: 'm', properties: ['top'] },
-  h: { size: 'm', properties: ['left'] },
+  $v: { size: 'm', properties: ['top'] },
+  $h: { size: 'm', properties: ['left'] },
```

Comment:
unsure why we had `type` and `as` so let me know if it should be there; it was erroring.

Reply:
Possibly it's needed. The default for button type is 'submit' and I know we've run into issues submitting forms unintentionally before, but not sure if that would be the case with this one.

Reply:
OK good shout, I'll add it as an expected prop:

```
const CloseButton = styled(Space).attrs<{ type: string }>({
  as: 'button',
  type: 'button',
  $v: { size: 'm', properties: ['top'] },
  $h: { size: 'm', properties: ['left'] },
})`
```